### PR TITLE
use postal code stuff in sseding

### PR DESCRIPTION
**Pull request by aspensantos**

### CODE REVIEW

File: internal/database/migrations/migrations.go [hunk 1]
```diff
@@ -92,13 +92,13 @@ func Seed(db *gorm.DB, locationFinder location.LocationFinder) error {
 	}
 
 	// set lat long for coordinates
-	for _, story := range stories {
-		coordinates, err := locationFinder.GetCityCenter(story.CurrentCity)
+	for index, story := range stories {
```
Comment (nimwijetunga) [inline internal/database/migrations/migrations.go:95]: Can you run `make seed` and make sure this works and is seeded properly

File: internal/location/coordinates.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func (finder coordinatesFinder) randomizePinLocation(city string) (latitude floa
 
 func (finder coordinatesFinder) GetPostalLatitudeAndLongitude(city string, limit int64) (float64, float64, error) {
 	// query zipcode with numStoriesInCity + 1 results and take the last one
-	resp, err := http.Get(fmt.Sprintf(zipCodeURL, finder.zipCodeToken, city, limit+1))
+	resp, err := http.Get(fmt.Sprintf(zipCodeURL, finder.zipCodeToken, city, limit+2))
```
Comment (nimwijetunga) [inline internal/location/coordinates.go:88]: Did you test that this now works?

**Review by nimwijetunga — CHANGES_REQUESTED** [review]
Mostly just testing

File: internal/database/migrations/migrations.go (hunk 1 above)
Comment (aspensantos) [reply internal/database/migrations/migrations.go:95]: yeah i ran it and it worked

File: internal/location/coordinates.go (hunk 2 above)
Comment (aspensantos) [reply internal/location/coordinates.go:88]: yeah, tested

**Review by nimwijetunga — APPROVED** [review]
LGTM

**Pull request merged**